src/main/java/org/opensearch/performanceanalyzer/commons/stats/ServiceMetrics.java

```diff
@@ -21,9 +21,7 @@
  */
 public class ServiceMetrics {
     public static SampleAggregator READER_METRICS_AGGREGATOR,
-            STAT_METRICS_AGGREGATOR = new SampleAggregator(StatMetrics.values()),
```

Comment:
Any specific reasons we removed those aggregators? Are those just for testing?

Reply:
These aggregators are unused in the entire Performance Analyzer codebase. They seem to be leftover from an earlier refactoring possibly involving RCA.

Do you think this is a change that would benefit from being it's own PR?